### ISSUE TITLE
Persistent data paths

### ISSUE THREAD

**Issue by mavhc** [jajm/koha-docker#6]
Do you have a list of which paths contain data that could be linked to a volume so this could be used for non testing purposes?

**Comment by jajm** [comment]
No, and be aware that these images were built for testing/dev only. IMO they shouldn't be used in production.

**Comment by corgijan** [comment]
I think there is an opportunity to make the setup workflow a lot smoother by exposing them and having a solid docker build. Also the builds are currently broken @jajm. Are you interested in this @mavhc ?
